src/main.rs

```diff
@@ -260,6 +260,7 @@ extern crate lazy_static;
 #[cfg(test)]
 #[cfg(feature = "integration_tests")]
 mod tests {
+
```

Comment:
You should drop the whitespace changes in this file.

Reply:
Sure.